### DISCUSSION TITLE
Welcome to FATE-Flow Discussions!

### DISCUSSION THREAD

**Discussion by WebankAITeam** [FederatedAI/FATE-Flow#144] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wrwahut** [comment]
![image](https://user-images.githubusercontent.com/35087858/147055843-dfd74c7b-e28c-42a3-9a85-9135caf22e08.png)
